shell/platform/glfw/public/flutter_glfw.h

```diff
@@ -167,6 +167,17 @@ FLUTTER_EXPORT void FlutterDesktopWindowSetPixelRatioOverride(
     FlutterDesktopWindowRef flutter_window,
     double pixel_ratio);
 
+// Sets the min/max width/height of |flutter_window| in pixels. Use
+// GLFW_DONT_CARE to leave a dimension unconstrained.
+//
+// This must be called from the main thread.
```

Comment:
That's true of everything in this API that doesn't say otherwise; no need to explicitly state it for this function (doing so only here I think suggests that other functions could be called in any thread). You could add a file-level comment to that effect instead so it's clear.